oregon-a/basket-dev/ns.yaml

```diff
@@ -1,13 +1,7 @@
 apiVersion: v1
 kind: Namespace
 metadata:
-  creationTimestamp: "2018-03-07T16:20:03Z"
-  labels:
-    heritage: deis
   name: basket-dev
-  resourceVersion: "3197798"
-  selfLink: /api/v1/namespaces/basket-dev
-  uid: 645beaf8-2223-11e8-8262-02e6188a289c
 spec:
   finalizers:
   - kubernetes
```

Comment:
I don't believe we need the spec or status sections either, based on https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#creating-a-new-namespace